### PR TITLE
User Story 91050: Add quick-links to application form view

### PR DESCRIPTION
**Pull request by aishajilani**

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ApplicationForm/ApplicationFormIntegrationTests.cs [hunk 1]
```diff
@@ -257,5 +257,94 @@ public async void Should_Show_500Error_When_Application_Is_Not_Valid()
...
+
+			Document.QuerySelectorAll("#Declaration_link")
+				.Where(contents => contents.InnerHtml == "Declaration")
+				.Should()
+				.NotBeEmpty();
+			Document.QuerySelectorAll("#Declaration_link")
+				.First().Attributes.Where(a => a.Value == "#Declaration")
+				.Should()
+				.NotBeEmpty();
+		}
```
Comment (CathLass) [inline ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ApplicationForm/ApplicationFormIntegrationTests.cs:348]: I think we'd have to test that the id is attached to the section header too. I had a stab at it below, but there might be a more direct way to do it
```
			// check the link text and href
			var sectionLink = Document.QuerySelector("#Overview_link");
			sectionLink.TextContent.Should().Be("Overview");
			sectionLink.Attributes
				.Where(a => a.Name == "href")
				.First().Value
				.Should().Be("#Overview");
			// check the link is associated with the correct section header
			Document.QuerySelectorAll("h2")
				.Where(contents => contents.InnerHtml == "Overview")
				.First().Attributes.Where(a => a.Name == "id")
				.First().Value
				.Should().Be("Overview");
```

**Review by CathLass — CHANGES_REQUESTED** [review]

File: ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ApplicationForm/ApplicationFormIntegrationTests.cs [hunk 2]
```diff
@@ -257,5 +257,94 @@ public async void Should_Show_500Error_When_Application_Is_Not_Valid()
 			Document.QuerySelector("#error-heading").Should().NotBeNull();
 			Document.QuerySelector("#error-heading").TextContent.Should().Contain("Internal server error");
 		}
+
+		[Fact]
+		public async void Validate_Quicklinks_On_SchoolApplicationForm()
```
[the remark this answers is not in the document]
Comment (aishajilani) [reply ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Pages/ApplicationForm/ApplicationFormIntegrationTests.cs:262]: Changed to - Should_Contain_Contents_With_Links_To_Correct_Sections()

File: ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ApplicationForm/Index.cshtml [hunk 3]
```diff
@@ -9,14 +9,27 @@
...
+		}
+	</ol>
+</nav>
 
 <div class="govuk-grid-row">
 	<div class="govuk-grid-column-full">
 		@foreach (var formSection in Model.Sections)
 		{
-			<h2 class="govuk-heading-l govuk-!-margin-bottom-2 govuk-!-font-size-36">@formSection.Heading</h2>
-			<p class="govuk-body-s"><a asp-page="@Links.SchoolApplicationForm.Index.Page" asp-route-id="@Model.Project.Id" target="application"  id="@("application-form-link")">Open school application form in a new tab</a></p>
```
[the remark this answers is not in the document]
Comment (aishajilani) [reply ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ApplicationForm/Index.cshtml:19]: Added back on.

**Review by CathLass — APPROVED** [review]
looks good

**Review by kolevk — APPROVED** [review]

**Pull request closed without merging**